lib/puppet/type/yumrepo.rb

```diff
@@ -304,19 +304,25 @@
 
   newproperty(:proxy) do
     desc "URL of a proxy server that Yum should use when accessing this repository.
-      This attribute can also be set to `'_none_'`, which will make Yum bypass any
+      This attribute can also be set to `'_none_'` or `''`(empty string), which will make Yum bypass any
       global proxy settings when accessing this repository.
       #{ABSENT_DOC}"
 
     newvalues(%r{.*}, :absent)
     validate do |value|
-      next if value.to_s =~ %r{^(absent|_none_)$}
+      next if value.to_s =~ %r{^(absent|_none_|)$}
```

Comment:
Maybe it would be a good idea for the code readability and consistency to replace the last 'or' with a line such as:
`next if value.to_s.empty?`